cpp/include/cudf/strings/strings_column_view.hpp

```diff
@@ -128,21 +131,22 @@ class strings_column_view : private column_view {
    * The offsets child must be used to properly address the char bytes.
    *
    * For example, to access the first character of string `i` (accounting for
-   * a sliced column offset) use: `chars_begin()[offsets_begin()[i]]`.
+   * a sliced column offset) use: `chars_begin(stream)[offsets_begin()[i]]`.
    *
    * @return Iterator pointing to the first char byte.
    */
-  [[nodiscard]] chars_iterator chars_begin() const;
+  [[nodiscard]] chars_iterator chars_begin(rmm::cuda_stream_view) const;
```

Comment:
No name parameter? So it actually doesn't need a stream, right? Just added for consistency?

Reply:
Mostly for consistency but also for possible future use.